OracleDataIntegrator/README.md

```diff
@@ -0,0 +1,138 @@
+
+At the end of this configuration there will be 2 containers running : 1) DB Container 2) ODI Agent.
+
+## Pre-Requisite
+
+1. Mount a host directory as a data volume
+
+Data volumes are designed to persist data, independent of the container’s lifecycle. he default location of the volume in container is under "/var/lib/docker/volumes".
+There is an option to mount a directory from your Docker engine’s host into a container as volume. In this project we will use that option for the data volume. 
+The volume will be used to store Database datafiles and ODI domain files.
+
+Since the volume is created as "root" user, provide read/write/execute permissions to "oracle" user (by providing permissions to "others"), as all operations inside the container happens with "oracle" user login.
+       $ mkdir /scratch/DockerVolume/ODIVolume/DB
+       $ mkdir /scratch/DockerVolume/ODIVolume/ODI
+       $ chmod -R 777 /scratch/DockerVolume/ODIVolume
+
+2. Database
+
+You need to have a running database container or a database running on any machine. 
+The database connection details are required for creating ODI specific RCU schemas while configuring ODI domain. 
+While using a 12.2.0.1 CDB/PDB DB, ensure PDB is used to load the schemas. RCU loading on CDB is not supported.
+
+The Oracle Database image can be pulled from the [Docker Store](https://store.docker.com/images/oracle-database-enterprise-edition) or the [Oracle Container Registry](https://container-registry.oracle.com) or you can build your own using the Dockerfiles and scripts in [GitHub Location](https://github.com/oracle/docker-images/tree/master/OracleDatabase/dockerfiles/12.2.0.1)
+
+Create an environment file **db.env.list**
+
+        ORACLE_SID=<DB SID>
+        ORACLE_PDB=<PDB ID>
+        ORACLE_PWD=<password>
+        
+Sample Data will look like this...
+
+	ORACLE_SID=odidb
+        ORACLE_PDB=odipdb
+        ORACLE_PWD=Welcome1
+        
+Sample Command to Start the Database is as follows
+
+         $ docker run --name ODI122126Database  -p 1521:1521 -p 5500:5500 -v /scratch/DockerVolume/ODIVolume/DB:/opt/oracle/oradata --env-file ./db.env.list  oracle/database:12.2.0.1-ee
+
+The above command starts a DB container mounting a host directory as /opt/oracle/oradata for persistence. 
+It maps the containers 1521 and 5500 port to respective host port such that the services can be accessible outside of localhost.
+
+## ODI Distributable version 12.2.1.2.6 Docker image Creation and Running
+
+To build a ODI image either you can start from building Oracle JDK or use the already available Oracle JDK image.
+The Oracle JDK  image is not currently available.
+If plan to use available Oracle JDK image, skip next steps and jump to "Building Docker Image for ODI"
```

Comment:
Change this to: "If _you_ plan to use _the_ available Oracle JDK image, skip the next step and continue with "Building the ODI Docker Image"